rdbms/src/main/java/io/spine/server/storage/jdbc/JdbcStandStorage.java

```diff
@@ -0,0 +1,268 @@
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.server.storage.jdbc;
+
+import com.google.common.base.Converter;
+import com.google.common.base.Function;
+import com.google.common.base.Optional;
+import com.google.common.base.Predicate;
+import com.google.common.collect.Iterators;
+import com.google.protobuf.Any;
+import com.google.protobuf.FieldMask;
+import io.spine.client.EntityFilters;
+import io.spine.client.EntityId;
+import io.spine.client.EntityIdFilter;
+import io.spine.server.entity.AbstractVersionableEntity;
+import io.spine.server.entity.Entity;
+import io.spine.server.entity.EntityRecord;
+import io.spine.server.entity.storage.EntityQueries;
+import io.spine.server.entity.storage.EntityQuery;
+import io.spine.server.entity.storage.EntityRecordWithColumns;
+import io.spine.server.stand.AggregateStateId;
+import io.spine.server.stand.StandStorage;
+import io.spine.string.Stringifier;
+import io.spine.string.StringifierRegistry;
+import io.spine.type.TypeUrl;
+
+import javax.annotation.Nullable;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static com.google.common.base.Preconditions.checkState;
+import static com.google.common.collect.Iterables.transform;
+import static com.google.common.collect.Iterators.filter;
+import static com.google.common.collect.Lists.newArrayList;
+import static io.spine.protobuf.TypeConverter.toAny;
+import static io.spine.util.Exceptions.illegalStateWithCauseOf;
+import static java.util.Collections.singleton;
+
+/**
+ * JDBC-based implementation of {@link StandStorage}.
+ *
+ * @author Dmytro Dashenkov
+ */
+public class JdbcStandStorage extends StandStorage {
+
+    private static final EntityQuery<String> EMPTY_QUERY = EntityQueries.from(
+            EntityFilters.getDefaultInstance(),
+            Entity.class
+    );
+
+    private static final Converter<AggregateStateId, String> ID_MAPPER;
+
+    private final JdbcRecordStorage<String> recordStorage;
```

Comment:
The declaration of this field may raise questions (due to `<String>`). Please have a comment.